analyses/AR_cd_2010/doc_AR_cd_2010.md

```diff
@@ -0,0 +1,18 @@
+# 2010 Arkansas Congressional Districts
+
+## Redistricting requirements
+In Arkansas, there are no state law requirements for congressional districts.
+
+### Interpretation of requirements
+We enforce a maximum population deviation of 0.5%, which is in line with the low deviation seen in past congressional district maps.
+We limit the number of county/municipality splits, which is in line with the small number of county/municipality splits observed in past congressional district maps.
+
+## Data Sources
+Data for Arkansas comes from the ALARM Project's [Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for Arkansas across two independent runs of the SMC algorithm.
+No special techniques were needed to produce the sample.
```

Comment:
Mention the county constraint here.

ALso can you copy this file into the top of the PR?

Reply:
Whoops, yes, copied over.
Arkansas doesn't have any county constraints.

Reply:
Hm, you're using a county constraint?
https://github.com/alarm-redist/fifty-states/pull/147/files#diff-430dd4ef6bd4d2cef276afc2caeafe34a7234bc34b003ea6ba2a9b3f62f45c14R10  
And you say so earlier in the documentation

It's good you're doing this! I just meant that here in the doc file you should mention you told the SMC algorithm to preserve counties.  The way I think about it is the 'interpretation' section is like how you parse the rules and the 'notes' section here is like how you implemented them in the algorithm.